src/config/setup-test-container.mjs

```diff
@@ -61,7 +61,7 @@ export function setupTestContainer() {
     getSimpleClient: awilix.asValue(
       stub().returns({ close: () => Promise.resolve() }),
     ),
-    accountClient: awilix.asFunction(stub()),
+    AccountClient: awilix.asValue(stub()),
```

Comment:
this'll probably break tests - the code under test will call the stub to construct an accountClient, and then it'll call stub.startOAuthRequest, stub.getToken, and stub.getSession. what about something like:

```suggestion
    AccountClient: awilix.asValue(() => ({ startOAuthRequest: stub(), getToken: stub(), getSession: stub() })),
```

Reply:
i see the test code using mockAccountClient - if it's a good default mock, maybe we move it to here so the tests don't have to individually register it?